Mathlib/AlgebraicGeometry/GammaSpecAdjunction.lean

```diff
@@ -301,6 +301,8 @@ def identityToΓSpec : 𝟭 LocallyRingedSpace.{u} ⟶ Γ.rightOp ⋙ Spec.toLoc
       dsimp [toΓSpecBase]
       rw [ContinuousMap.coe_mk, ContinuousMap.coe_mk]
       dsimp [toΓSpecFun]
+      -- note to reviewers: this was found before the refactor
```

Comment:
What do you mean with this note?
Why is the `have` now needed?

Reply:
that the instance did work before the refactor and was thus broken by this refactor

Reply:
I'm not sure why it breaks now, either